living-documents/my-guide-to-the-twin-cities.md

```diff
@@ -76,7 +76,7 @@ Places for drinking alcohol and/or coffee/tea.
 - [Minnehaha Falls](https://www.nps.gov/miss/planyourvisit/minnehah.htm) - A water fall right in the city.
 - [Lake Superior](https://en.wikipedia.org/wiki/Lake_Superior) - The largest of North America's "Great Lakes", and the largest freshwater lake in the world by surface area. This is about a 3 hour drive from the city, but still worth seeing if you can make the trip.
 - [Afton Alps](https://www.aftonalps.com) - There are a few ski resorts near the Twin Cities, but Afton is the closest.
-- [Chain of Lakes](https://www.minneapolisparks.org/parks__destinations/parks__lakes/minneapolis_chain_of_lakes_regional_park/) - Possibly Minneapolis' most unique feature is its lake system *within the city* ([David Byrne](https://en.wikipedia.org/wiki/David_Byrne) is supposedly a big fan). The chain consists of Cedar Lake, Lake of the Isles, Lake Calhoun, and Lake Harriet (my favorite of the lakes). These are all connected by bike trails (see the Grand Rounds below), and also by waterways, allowing them to be traversed by canoe.
+- [Chain of Lakes](https://www.minneapolisparks.org/parks__destinations/parks__lakes/minneapolis_chain_of_lakes_regional_park/) - Possibly Minneapolis' most unique feature is its lake system *within the city* ([David Byrne](https://en.wikipedia.org/wiki/David_Byrne) is supposedly a big fan). The chain consists of Cedar Lake, Lake of the Isles, Bde Maka Ska ([formerly Lake Calhoun](https://en.wikipedia.org/wiki/Bde_Maka_Ska)), and Lake Harriet (my favorite of the lakes). These are all connected by bike trails (see the Grand Rounds below), and also by waterways, allowing them to be traversed by canoe.
```

Comment:
Good catch! Let's just leave the link off since none of the others are linked. Might go through and add links later. :)

```suggestion
- [Chain of Lakes](https://www.minneapolisparks.org/parks__destinations/parks__lakes/minneapolis_chain_of_lakes_regional_park/) - Possibly Minneapolis' most unique feature is its lake system *within the city* ([David Byrne](https://en.wikipedia.org/wiki/David_Byrne) is supposedly a big fan). The chain consists of Cedar Lake, Lake of the Isles, Bde Maka Ska (formerly Lake Calhoun), and Lake Harriet (my favorite of the lakes). These are all connected by bike trails (see the Grand Rounds below), and also by waterways, allowing them to be traversed by canoe.
```